www/src/index.ts

```diff
@@ -1,14 +1,16 @@
-import m from 'mithril';
-import Home from './views/home';
-import About from './views/about';
-import Register from './views/register';
-import Profile from './views/profile';
-import Statistics from './views/statistics';
+import m from "mithril";
+import home from "./views/home";
+import about from "./views/about";
+import register from "./views/register";
+import profile from "./views/profile";
+import statistics from "./views/statistics";
+import login from "./views/login";
 
-m.route(document.body, '/', {
-  '/': Home,
-  '/about': About,
-  '/register': Register,
-  '/profile': Profile,
-  '/statistics': Statistics,
+m.route(document.body, "/", {
+    "/": home,
```

Comment:
Are you using a formatter that changes to `"` from `'`? Because my formatter does the opposite :hankey:

Reply:
I'll create a ticket for a lint/formatter command with `npm run lint` and `npm run format`.